### ISSUE TITLE
[Feature request] Display tabs from other devices

### ISSUE THREAD

**Issue by WallK** [perfect-things/perfect-home#157]
Hello! Thanks for the great extension, it makes my life much easier

On chrome I've used a different home page extension that did pretty much the same thing, but not as pretty (:

It also had a feature I would really like to have in Perfect Home on Firefox, if at all possible: display a list of tabs from other devices

It makes jumping from device to device one click deal and should enhance the experience greatly

Thanks!

**Comment by tborychowski** [comment]
Thanks for the feedback.
Unfortunately this isn't possible at the moment. 
Extensions don't have access to user's accounts' data.
The only way to achieve that would be to build extensions for mobile and desktop and make users create accounts with some sort of a cloud to store your data and sync them based on your account, which is wrong on so many levels that I wouldn't even want to go there :-)

**Comment by WallK** [comment]
Oh, that's a shame! I guess Chrome has a very different access policies
You are right, this is not a way forward (:
I guess I could always bookmark what I want and delete it afterward (or, more likely, forget about it forever! (: )
Thank you!

**Comment by tborychowski** [comment]
Yeah :-) I understand your predicament.
I tried to use cloud-tabs on my apple devices (safari has this built-in), but it wasn't working to 100% and it wasn't instant. So I gave up.

I moved to a task manager app, so for every tab (or rather URL) that I need to do something about later or on another device - I add a task and close the tab :-)
Not sure if this will fit your workflow.

Lastly, Firefox has something like this: 
https://support.mozilla.org/en-US/kb/send-tab-firefox-desktop-other-devices
